akka-actor/src/main/scala/akka/dispatch/affinity/AffinityPool.scala

```diff
@@ -152,14 +236,12 @@ private[akka] class AffinityPool(
     }
 
   override def execute(command: Runnable): Unit = {
-    if (command == null)
```

Comment:
tryEnqueue would lead to an NPE if command was null, so in essence we were nullchecking twice.